scheduler/domain/README.md

```diff
@@ -5,7 +5,7 @@ API, and distributing the jobs to workers and maintaining and communicating
 the state of jobs run.
 
 The scheduler code contains the following packages:
-* __sched__ - scheduler go objects: jobs, tasks and states, and thrift versions of these objects
+* __domain__ - scheduler go objects: jobs, tasks and states, and thrift versions of these objects
   * __gen-go/sched__ - generated code from the thrift definitions
   * __scheduler__ - interfaces and implementations for job scheduling
   * __worker__ - interface for scheduler to run tasks on a worker
```

Comment:
this is a relatively simplistic statement that always confused me... is it more accurate to say: starting tasks on workers, and tracking and managing responses from workers.  (not just 'run tasks on a worker')